### ISSUE TITLE
Unable to destroy activity: java.lang.NullPointerException

### ISSUE THREAD

**Issue by IvanDeluxe** [robodigital/cobrowseio_flutter#6]
Hi,

So I've implemented this package yesterday & within the first couple of initializations **in production** I've got this error @ sentry:

```
java.lang.NullPointerException: Attempt to invoke virtual method 'void io.flutter.plugin.common.i.e(io.flutter.plugin.common.i$c)' on a null object reference
    at i.a.a.a.b.onDetachedFromEngine(CobrowseIOFlutterPlugin.java:1)
    at io.flutter.embedding.engine.d.w(FlutterEngineConnectionRegistry.java:24)
    at io.flutter.embedding.engine.d.x(FlutterEngineConnectionRegistry.java:2)
    at io.flutter.embedding.engine.d.y(FlutterEngineConnectionRegistry.java:1)
    at io.flutter.embedding.engine.d.l(FlutterEngineConnectionRegistry.java:3)
    at io.flutter.embedding.engine.b.e(FlutterEngine.java:4)
    at io.flutter.embedding.android.f.t(FlutterActivityAndFragmentDelegate.java:14)
    at io.flutter.embedding.android.e.onDestroy(FlutterActivity.java:4)
    at android.app.Activity.performDestroy(Activity.java:8571)
    at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1364)
    at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5937)
    at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5995)
    at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:47)
    at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
    at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loopOnce(Looper.java:226)
    at android.os.Looper.loop(Looper.java:313)
    at android.app.ActivityThread.main(ActivityThread.java:8663)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)

java.lang.NullPointerException: Attempt to invoke virtual method 'void io.flutter.plugin.common.i.e(io.flutter.plugin.common.i$c)' on a null object reference
    at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5950)
    at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5995)
    at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:47)
    at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
    at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loopOnce(Looper.java:226)
    at android.os.Looper.loop(Looper.java:313)
    at android.app.ActivityThread.main(ActivityThread.java:8663)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
```

I honestly don't know how to reproduce this, but as soon as I find out something, I'll provide updates.

Thanks for this amazing package btw ❤

**Comment by IvanDeluxe** [comment]
Don't know if it helps, but it happened @ Galaxy S21, Android 12.

Also, flutter doctor output:
```
[√] Flutter (Channel stable, 2.10.4, on Microsoft Windows [Version 10.0.19043.1586], locale hr-HR)
    • Flutter version 2.10.4 at C:\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision c860cba910 (11 days ago), 2022-03-25 00:23:12 -0500
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at C:\Users\ivan7\AppData\Local\Android\sdk
    • Platform android-31, build-tools 29.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Build Tools 2019 16.11.5)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.31729.503
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 4.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.66.0)
    • VS Code at C:\Users\ivan7\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.38.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19043.1586]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 100.0.4896.60
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 100.0.1185.29

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by robinbonnes** [comment]
Thanks, should be fixed in the latest version (1.0.5)
